docs/source/changelog.rst

```diff
@@ -8,10 +8,31 @@ Current version
 Changes
 -------
 
-* Remove support for Python 3.6 and 3.7; add support for up to Python 3.11.
+* Remove support for Python 3.6 and 3.7; add support for up to Python 3.11
+  (`#154 <https://github.com/jensengroup/propka/issues/154>`_ and
+  `#150 <https://github.com/jensengroup/propka/pull/150>`_)
 
-v3.4.0
-======
+* Add context manager for ``open_file_for_reading``
+  (`#133 <https://github.com/jensengroup/propka/pull/133>`_)
+
+Fixes
+-----
+
+* Refactor and fix ``MolecularContainer.get_pi()``
+  (`#148 <https://github.com/jensengroup/propka/pull/148>`_)
+
+* Rename vanadium from ``Va`` to ``V``
+  (`#141 <https://github.com/jensengroup/propka/pull/141>`_)
+
+* Fix rounding issues
```

Comment:
I suggest to be a bit more specific: "Fix rounding issues in folding profile reporting"

Reply:
Fixed in c74e7cda665637f334881f3e2434dca7d365b818.